### ISSUE TITLE
Did you figure out hcaptcha?

### ISSUE THREAD

**Issue by delebash** [HyperionCSharp/EpicGamesAPIDocs#10]
You posted about it on the epic-freegames-node app.  https://github.com/claabs/epicgames-freegames-node/issues/221

Did you ever figure it out?

Currently I am using fiddler and logging in with my launcher to grab the auth key.  However, I would like to get rid of fiddler.  I do not mind solving the hcaptcha manually, but I still don't know how to get to that point.

I am using this to get unreal engine vault information provided by the epic game's launcher.

My current app https://github.com/delebash/unreal_vault_organizer

TY!

**Comment by HyperionCSharp** [comment]
Yes, you have to use my fingerprint gen to generate a valid fingerprint so epic can verify your captcha.

**Comment by delebash** [comment]
cool, ty

**Comment by delebash** [comment]
Do you have a workflow for authenticating unreal launcher.  I see your fingerprint generator, but I am not sure what to do with it.  Does it have to do with generating the xal?
